Operations_on_polyhedra/include/CGAL/internal/Polyhedron_plane_clipping_3.h

```diff
@@ -131,7 +131,7 @@ Polyhedron clip_to_bbox(const Bbox_3& bbox, const Plane_3& plane)
     if (orientations[i2]==ON_ORIENTED_BOUNDARY) continue;
     if (orientations[i1]!=orientations[i2])
       intersection_points.push_back(
-        get<Point_3>(
+        boost::get<Point_3>(
```

Comment:
That's a work around for a compiler bug. The argument of get is a `boost::variant` so ADL should work but it doesn't.